packages/babel-plugin-open-source/babel.js

```diff
@@ -82,9 +82,10 @@ module.exports = declare((api) => {
         url = `vscode://file/${state.filename}:${location.start.line}`;
       }
       const sourceData = JSON.stringify({ url });
+      const sourceNode = t.jsxExpressionContainer(t.templateLiteral([t.templateElement({ raw: sourceData, cooked: sourceData }, true)], []))
```

Comment:
This is a good idea! 

Tiny changes: don't need to pass cooked or tail here.

```suggestion
      const sourceNode = t.jsxExpressionContainer(t.templateLiteral([t.templateElement({ raw: sourceData })], []))
```